Python/bytecodes.c

```diff
@@ -3808,10 +3809,7 @@ dummy_func(
         }
 
         op(EXIT_TRACE, (--)) {
-            frame->prev_instr--;  // Back up to just before destination
-            _PyFrame_SetStackPointer(frame, stack_pointer);
-            Py_DECREF(self);
-            return frame;
+            goto deoptimize;
```

Comment:
In uops debug mode (`PYTHONUOPSDEBUG=2` or higher) this would (confusingly, IMO) print `DEOPT: [Opcode ...]`. Doesn't that bother you? Or do you consider `EXIT_TRACE` as just another form of deoptimization?

Reply:
It definitely is annoying. Actual deoptimizations are rare, but exiting a trace is common. So I get a lot of log spam containing endless repetitions of
```
DEOPT: [Opcode 300, operand 0]
```

Reply:
This is tier 2 only, so it doesn't matter much what the implementation is. I can see it changing quite a few times in the near future.
Whatever it is, it shouldn't have the context dependent `frame->prev_instr--;  // Back up to just before destination`